plugins/web/opentelemetry-instrumentation-document-load/README.md

```diff
@@ -82,6 +82,29 @@ Because the browser does not send a trace context header for the initial page na
 </body>
 ```
 
+## Optional : Add custom attributes to document load span if needed
+
+If it is needed to add custom attributes to the document load span,and/or document fetch span and/or resource fetch spans, respective functions to do so needs to be provided
+as a config to the DocumentLoad Instrumentation as shown below.
+
+```js
+
+const addCustomAttributesToSpan = (span: Span) => {
+  span.setAttribute('<custom.attribute.key>','<custom-attribute-value>');
+}
+
+registerInstrumentations({
+  instrumentations: [
+    // @ts-ignore
+    new DocumentLoadInstrumentation({
+      applyCustomAttributesOnDocumentLoadSpan: addCustomAttributesToSpan,
```

Comment:
Can you document specifically when each of these is called? Is it the start or the end of the span? Somewhere in the middle?

Reply:
Also documenting the behavior if the custom attribute callback throws